src/patients/new/NewPatientForm.tsx

```diff
@@ -274,7 +278,9 @@ const NewPatientForm = (props: Props) => {
             />
           </div>
         </div>
-
+        {errorMessage && (
+          <div className="alert alert-danger" role="alert">{t(errorMessage)}</div>
```

Comment:
I think that error messages should be above the form fields. @lauggh @StefanoMiC thoughts here?

Reply:
+1